engine.py

```diff
@@ -39,6 +39,8 @@ def __init__(self):
 						"b":self.get_bishop_moves, "n":self.get_knight_moves}
 		self.light_king_pos = (7,4) # light king startung row and column
 		self.dark_king_pos = (0,4) # dark king starting row and column
```

Comment:
I think it will be better to use a dictionary for this
self.king_pos = {"kl":(7,4), "kd":(0,4)